site/content/en/docs/tasks/administer_cluster_quotas.md

```diff
@@ -25,10 +25,11 @@ In the following steps, you will create a queuing system with a single
 ClusterQueue and a single [ResourceFlavor](/docs/concepts/cluster_queue#resourceflavor-object)
 to govern the quota of your cluster.
 
-You can perform all these steps at once by applying [github.com/kubernetes-sigs/kueue/blob/main/examples/single-clusterqueue-setup.yaml](https://github.com/kubernetes-sigs/kueue/blob/main/examples/single-clusterqueue-setup.yaml):
+You can perform all these steps at once by
+applying [github.com/kubernetes-sigs/kueue/blob/main/examples/admin/single-clusterqueue-setup.yaml](https://github.com/kubernetes-sigs/kueue/blob/main/examples/admin/single-clusterqueue-setup.yaml):
```

Comment:
I think this link will not work as the current `https://github.com/kubernetes-sigs/kueue/blob/main/examples/single-clusterqueue-setup.yaml` does not work. Only this works with `site/static`: `https://github.com/kubernetes-sigs/kueue/blob/main/site/static/examples/single-clusterqueue-setup.yaml`. 

Please fix the links in this PR, or open another prep or follow up PR to make sure the link works. I'm also ok with replacing the link with the include.